conda-recipe/run_test.bat

```diff
@@ -28,54 +28,45 @@ if "%PYTHON%"=="python" (
     set NO_DIST=1
 )
 
+
+
 %PYTHON% -c "from sklearnex import patch_sklearn; patch_sklearn()" || set exitcode=1
 
+set "PYTEST_ARGS= "
+
+IF DEFINED COVERAGE_RCFILE (set PYTEST_ARGS=--cov=onedal --cov=sklearnex --cov-config="%COVERAGE_RCFILE%" --cov-append --cov-report= %PYTEST_ARGS%)
+
 rem Note: execute with argument --json-report as second argument
 rem in order to produce a JSON report under folder '.pytest_reports'.
-set with_json_report=0
 if "%~2"=="--json-report" (
-    set with_json_report=1
+    set PYTEST_ARGS=--json-report --json-report-file=.pytest_reports\FILENAME.json %PYTEST_ARGS%
     mkdir .pytest_reports
     del /q .pytest_reports\*.json
 )
 
 echo "NO_DIST=%NO_DIST%"
-echo "with_json_report=%with_json_report%"
 setlocal enabledelayedexpansion
-if "%with_json_report%"=="1" (
-    pytest --verbose -s "%1tests" --json-report --json-report-file=.pytest_reports\legacy_report.json || set exitcode=1
-    pytest --verbose --pyargs daal4py --json-report --json-report-file=.pytest_reports\daal4py_report.json || set exitcode=1
-    pytest --verbose --pyargs sklearnex --json-report --json-report-file=.pytest_reports\sklearnex_report.json || set exitcode=1
-    pytest --verbose --pyargs onedal --json-report --json-report-file=.pytest_reports\onedal_report.json || set exitcode=1
-    pytest --verbose "%1.ci\scripts\test_global_patch.py" --json-report --json-report-file=.pytest_reports\global_patching_report.json || set exitcode=1
-    if NOT "%NO_DIST%"=="1" (
-        %PYTHON% "%1tests\helper_mpi_tests.py"^
-            pytest -k spmd --with-mpi --verbose -s --pyargs sklearnex^
-            --json-report --json-report-file=.pytest_reports\sklearnex_spmd.json
-        if !errorlevel! NEQ 0 (
-            set exitcode=1
-        )
-        %PYTHON% "%1tests\helper_mpi_tests.py"^
-            pytest --with-mpi --verbose -s "%1tests\test_daal4py_spmd_examples.py"^
-            --json-report --json-report-file=.pytest_reports\mpi_legacy.json
-        if !errorlevel! NEQ 0 (
-            set exitcode=1
-        )
+pytest --verbose -s "%1tests" %PYTEST_ARGS:FILENAME=legacy_report%|| set exitcode=1
```

Comment:
I'm not sure here if having a space before `||` makes a difference or not, but I think it'd be better to leave it just in case, just for clarity that it's not part of the argument.

Reply:
Sounds good, I will let CI continue running to verify things as they are and will make changes combined with whatever failures pop up.